staging/src/k8s.io/client-go/examples/leader-election/main.go

```diff
@@ -56,21 +55,17 @@ func buildConfig(kubeconfig string) (*rest.Config, error) {
 func main() {
 	klog.InitFlags(nil)
 
+	var leaderElect bool
 	var kubeconfig string
 	var leaseLockName string
 	var leaseLockNamespace string
-	var id string
 
+	flag.BoolVar(&leaderElect, "leader-elect", true, "whether enable leader elect")
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "absolute path to the kubeconfig file")
-	flag.StringVar(&id, "id", "", "the holder identity name")
```

Comment:
Leave this flag but change it to:

```
flag.StringVar(&id, "id", uuid.New().String(), "the holder identity name")
```

Reply:
is it good to point people in the direction of automatic random identities for leader-elected controllers, or should we leave it requiring an id provided via invocation?

Reply:
I guess this lets them run n replicas with a single invocation